src/serac/numerics/functional/tuple_tensor_dual_functions.hpp

```diff
@@ -1,8 +1,16 @@
+// Copyright (c) 2019-2023, Lawrence Livermore National Security, LLC and
```

Comment:
2024

Reply:
The rest of the code hasn't been updated. I'll push a branch now. I was hoping to catch a point where there wasnt large PRs out but it's almost may.